.gitignore

```diff
@@ -2,8 +2,10 @@
 *.sublime-project
 *.sublime-workspace
 *.DS_Store
+.idea/
```

Comment:
why this?

Reply:
The IntelliJ IDEA IDEs create a folder to store all of the project's metadata, but you don't want it to be added to the repository in general, for various reasons (including privacy).

Reply:
You should not commit this

Reply:
Ok! I use to do this, but maybe that's not the ideal way to deal with this folder.